tests/tests/test_monitor_client.py

```diff
@@ -1187,10 +1189,84 @@ def test_monitorclient_logs_and_services(self, monitor_commercial_setup_no_clien
         assert fds_count1.rstrip().isnumeric()
         fds_count1 = int(fds_count1.rstrip())
         logger.info(
-            "test_monitorclient_alert_email: currently %s fds open" % fds_count0
+            "test_monitorclient_logs_and_services: currently %s fds open" % fds_count0
         )
         assert abs(fds_count1 - fds_count0) <= max_fds_count_diff
-        logger.info("test_monitorclient_alert_email: fds count have not increased")
+        logger.info(
+            "test_monitorclient_logs_and_services: fds count have not increased"
+        )
+
+    def test_monitorclient_logs_and_patterns(self, monitor_commercial_setup_no_client):
+        """Tests the monitor client email alerting for multiple patterns types"""
```

Comment:
It says "multiple patterns types" but I only see one type: PCRE

Reply:
true. what a lie :) I will correct this.